apis/infix-en_GB/atrium-api-infix-en_GB-jvm/src/module/module-info.java

```diff
@@ -1,6 +1,7 @@
 module ch.tutteli.atrium.api.infix.en_GB {
     requires ch.tutteli.atrium.domain.builders;
     requires kotlin.stdlib;
+    requires java.base;
```

Comment:
also add
```
    requires ch.tutteli.kbox;
    requires ch.tutteli.niok;
```
unfortunately there is a bug in Kotlin which does not detect this mistake :(

Reply:
also apply the same changes in apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/module/module-info.java